src/reference/asciidoc/dsl.adoc

```diff
@@ -1202,3 +1202,63 @@ That `errorRecovererFlow` can be used as follows:
 private Function<String, String> errorRecovererFlowGateway;
 ----
 ====
+
+[[java-dsl-extensions]]
+=== DSL Extensions
+
+Starting with version 5.3, an `IntegrationFlowExtension` has been introduced to allow to extend an existing Java DSL to any possible custom or composed EIP-operators.
+There is only enough to extend this class and write methods which are going to be used in the `IntegrationFlow` bean definitions.
```

Comment:
All that is needed is an extension of this class that provides methods which can be used in...